### ISSUE TITLE
ServerTimestamp returns as NoneType

### ISSUE THREAD

**Issue by LBeaufore** [FreeOpcUa/python-opcua#298]
When I attempt to access the ServerTimestamp of a node by using
`node.get_data_value().ServerTimestamp`
A "NoneType" value is returned. However, SourceTimestamp, val, and StatusCode seem to work in the place of ServerTimestamp. Is ServerTimestamp still supported in this way?

**Comment by oroulet** [comment]
Some servers do no set ServerTimestamp attribute

**Comment by LBeaufore** [comment]
I am almost certain this one does, it returned a server timestamp with Unified Automation's UaExpert GUI Client

**Comment by oroulet** [comment]
There's been an issue on this a few month ago, maybe try to update.
Otherwise you need to start wireshark and compare a session against python
and one against uaexpert and try to spot difference, not very difficult
since wireshark had built in support for opcua

**Comment by LBeaufore** [comment]
That makes sense, I'll try it out. Thank you!

**Comment by sanlop73** [comment]
Good afternoon:
Could you please give me the solution?
Getting the same issue here, Attached picture of UAexpert compare with
            print("SourceTimeStamp", val1.get_data_value().SourceTimestamp)
            print("ServerTimeStamp", val1.get_data_value().ServerTimestamp)

Thanks in advance

![image](https://user-images.githubusercontent.com/24301610/32945109-72c8c80a-cb92-11e7-904b-82992f8a2c5b.png)

**Comment by oroulet** [comment]
@sanlop73 record a wireshark session. you can also make use that we ask for both timestamps in query. I kind of remember this is part of read query

**Comment by sanlop73** [comment]
Good morning:
I have made your suggested test with Wireshark.
By default the library is asking for SourceTimestamp only, for read.
So i have modified the file Node.py
In function get_attribute, i have add 
**params.TimestampsToReturn = ua.TimestampsToReturn.Both**

Is this the right function to place the instruction?

Thanks in advance for your comments.


![read sent](https://user-images.githubusercontent.com/24301610/33005079-8563abcc-cdc4-11e7-97c0-f0e32a0c131e.png)
![read received](https://user-images.githubusercontent.com/24301610/33005085-88553a12-cdc4-11e7-8e73-049dc538dc33.png)

**Comment by sanlop73** [comment]
Sorry , forgot to upload the image

![timestamp](https://user-images.githubusercontent.com/24301610/33006043-193c7d0c-cdc9-11e7-899b-1e30979259f2.png)

**Comment by oroulet** [comment]
correct, It might be that we should have "both" as default

**Comment by sanlop73** [comment]
Yes, please, is better, you always have the chance to call or not to the variable.
Thanks
